File: factorial.java
public class factorial {
    int num;
    int fact=1;
    public void facto(int num){
        for (int i = 1; i <=num; i++) {
            fact = fact*i;
        }
        System.out.println("factorial is  " + fact );
    }
    public static void main(String[] args) {
        factorial obj = new factorial();
        obj.facto(5);
    }
}

File: factorial.cpp

#include <iostream>
using namespace std;

int main(){
    
    int num = 6,fact=1;

    
    for (int i = 1; i <= num; i++)
    {
        fact = fact*i;
    }
    cout << "Factorial is " << fact;
    return 0;
}